code/game/machinery/doors/multi_tile.dm

```diff
@@ -263,8 +263,21 @@
 		if (control.status != SHUTTLE_DOOR_BROKEN)
 			return ..()
 		if(!skillcheck(user, SKILL_ENGINEER, SKILL_ENGINEER_ENGI))
-			to_chat(user, SPAN_WARNING("You don't seem to understand how to restore a remote connection to [src]."))
-			return
+			if(!skillcheck(user, SKILL_PILOT, SKILL_PILOT_TRAINED))
+				to_chat(user, SPAN_WARNING("You don't seem to understand how to restore a remote connection to [src]."))
+				return
+			if(user.action_busy)
+				return
+			else
+				to_chat(user, SPAN_WARNING("You begin to restore the remote connection to [src]."))
+				if(!do_after(user, 8 SECONDS, INTERRUPT_ALL, BUSY_ICON_BUILD))
+					to_chat(user, SPAN_WARNING("You fail to restore a remote connection to [src]."))
+					return
+				unlock(TRUE)
+				close(FALSE)
+				control.status = SHUTTLE_DOOR_UNLOCKED
+				to_chat(user, SPAN_WARNING("You successfully restored the remote connection to [src]."))
+				return
```

Comment:
Your intends are wrong and code is working ONLY if you don't have engineering skill
```suggestion
		if(!skillcheck(user, SKILL_ENGINEER, SKILL_ENGINEER_ENGI) && !skillcheck(user, SKILL_PILOT, SKILL_PILOT_TRAINED))
			to_chat(user, SPAN_WARNING("You don't seem to understand how to restore a remote connection to [src]."))
			return
		if(user.action_busy)
			return
		else
			to_chat(user, SPAN_WARNING("You begin to restore the remote connection to [src]."))
			if(!do_after(user, 8 SECONDS, INTERRUPT_ALL, BUSY_ICON_BUILD))
				to_chat(user, SPAN_WARNING("You fail to restore a remote connection to [src]."))
				return
			unlock(TRUE)
			close(FALSE)
			control.status = SHUTTLE_DOOR_UNLOCKED
			to_chat(user, SPAN_WARNING("You successfully restored the remote connection to [src]."))
			return
```
```

Reply:
oh wait you just duplicated the code? Hold on

Reply:
going to rework it on another repo since i accidentally did it all on master instead of a branch. i didn't see another way to do the skill checks since you cant have more then 2 skills in a skill check (according to checks anyways)